sql/src/test/java/io/crate/planner/operators/LogicalPlannerTest.java

```diff
@@ -52,24 +53,37 @@ public void prepare() {
     private LogicalPlan plan(String statement) {
         SelectAnalyzedStatement analyzedStatement = sqlExecutor.analyze(statement);
         QueriedRelation relation = analyzedStatement.relation();
-        return LogicalPlanner.plan(relation)
+        return LogicalPlanner.plan(relation, FetchMode.WITH_PROPAGATION)
```

Comment:
Shouldn't this be `NO_PROPAGATION`? It is treated the same with these changes, did you want to test that?

Reply:
Doesn't really matter at this point because `WITH_PROPAGATION` isn't really implemented. (I'll probably get rid of the FetchMode altogether)